src/server/scripts/Events/hallows_end.cpp

```diff
@@ -57,13 +83,452 @@ class spell_hallows_end_has_water_bucket : public SpellScriptLoader
         }
     };
 
-    AuraScript* GetAuraScript() const
+    AuraScript* GetAuraScript() const override
     {
         return new spell_hallows_end_has_water_bucket_AuraScript();
     }
 };
 
+class spell_hallows_end_bucket_lands : public SpellScriptLoader
+{
+public:
+    spell_hallows_end_bucket_lands() : SpellScriptLoader("spell_hallows_end_bucket_lands") {}
+
+    class spell_hallows_end_bucket_lands_SpellScript : public SpellScript
+    {
+        PrepareSpellScript(spell_hallows_end_bucket_lands_SpellScript);
+
+        bool handled;
+        bool Load() { handled = false; return true; }
+        void HandleDummy(SpellEffIndex /*effIndex*/)
+        {
+            if (handled || !GetCaster())
+                return;
+
+            handled = true;
+            if (Player* target = GetHitPlayer())
+                GetCaster()->CastSpell(target, SPELL_CREATE_BUCKET, true);
+            else if (Unit* tgt = GetHitUnit())
+                GetCaster()->CastSpell(tgt, SPELL_WATER_SPLASH, true);
+        }
+
+        void Register() override
+        {
+            OnEffectHitTarget += SpellEffectFn(spell_hallows_end_bucket_lands_SpellScript::HandleDummy, EFFECT_0, SPELL_EFFECT_DUMMY);
+        }
+    };
+
+    SpellScript* GetSpellScript() const override
+    {
+        return new spell_hallows_end_bucket_lands_SpellScript();
+    }
+};
+
+class spell_hallows_end_base_fire : public SpellScriptLoader
+{
+public:
+    spell_hallows_end_base_fire() : SpellScriptLoader("spell_hallows_end_base_fire") { }
+
+    class spell_hallows_end_base_fire_AuraScript : public AuraScript
+    {
+        PrepareAuraScript(spell_hallows_end_base_fire_AuraScript);
+
+        void HandleEffectPeriodicUpdate(AuraEffect* aurEff)
```

Comment:
Since travis-ci seems to stop searching for unused parameters after the first it finds, see if you can spot any other missed unused parameters further down in the script (or fix this one and wait for the next travis-ci result).